tsconfig.json

```diff
@@ -15,6 +15,11 @@
     ],
     "exactOptionalPropertyTypes": true,
     "strictNullChecks": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "paths": {
```

Comment:
Again, we have some co-mingling here. Maybe we can either eliminate this by moving everything up one level as previously suggested, or maybe we can have a base tsconfig.json and then extend it separately for Cockpit and Insights?

Reply:
Unfortunately it was very hard to get away from this

Reply:
Sorry I initally added this comment to the wrong thread:
I know it's @croissanne's last day before PTO, but maybe she has some suggestions.